src/main/kotlin/no/nav/familie/klage/brev/baks/BaksBrevController.kt

```diff
@@ -0,0 +1,40 @@
+package no.nav.familie.klage.brev.baks
+
+import no.nav.familie.klage.felles.domain.AuditLoggerEvent
+import no.nav.familie.klage.infrastruktur.sikkerhet.TilgangService
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.UUID
+
+@RestController
+@RequestMapping(path = ["/api/baks/brev"])
+@ProtectedWithClaims(issuer = "azuread")
+@Validated
+class BaksBrevController(
+    private val baksBrevService: BaksBrevService,
+    private val tilgangService: TilgangService,
+) {
+    @GetMapping("/{behandlingId}/pdf")
+    fun hentBrev(
+        @PathVariable behandlingId: UUID,
+    ): Ressurs<ByteArray> {
+        tilgangService.validerTilgangTilPersonMedRelasjonerForBehandling(behandlingId, AuditLoggerEvent.ACCESS)
+        tilgangService.validerHarVeilederrolleTilStønadForBehandling(behandlingId)
+        return Ressurs.success(baksBrevService.hentBrev(behandlingId).pdfSomBytes())
+    }
+
+    @PostMapping("/{behandlingId}")
+    fun opprettEllerOppdaterBrev(
+        @PathVariable behandlingId: UUID,
+    ): Ressurs<ByteArray> {
+        tilgangService.validerTilgangTilPersonMedRelasjonerForBehandling(behandlingId, AuditLoggerEvent.UPDATE)
+        tilgangService.validerHarSaksbehandlerrolleTilStønadForBehandling(behandlingId)
```

Comment:
```suggestion
        tilgangService.validerHarSaksbehandlerRolleTilStønadForBehandling(behandlingId)
```
Nesten litt flau over at jeg nitpicker på dette, men det er bare fordi vi allerede har klasse som heter `SaksbehandlerRolle` haha

Reply:
Null stress, bare bra å ta slike ting! 😁 

Ser koden har følgende metoder: 

` 
   fun validerHarSaksbehandlerrolleTilStønadForBehandling(behandlingId: UUID) {
        validerHarRolleForBehandling(behandlingId, BehandlerRolle.SAKSBEHANDLER)
    }

    fun validerHarVeilederrolleTilStønadForBehandling(behandlingId: UUID) {
        validerHarRolleForBehandling(behandlingId, BehandlerRolle.VEILEDER)
    }

    fun validerHarVeilederrolleTilStønadForFagsak(fagsakId: UUID) {
        harTilgangTilFagsakGittRolle(fagsakId, BehandlerRolle.VEILEDER)
    }
`

`validerHarSaksbehandlerrolleTilStønadForBehandling` ser ut til å følge den samme strukturen som de andre, tenker du at man fortsatt burde endre på det?

Tenker uansett man kan endre på det i en egen PR da det ikke har så mye med denne PRen 😄

Reply:
Ja, for riktig camelcasing så bør det nok gjøres